### PR TITLE
Implement event handler

### PR DESCRIPTION
**Pull request by Bigizi**
**Math-Magician-Project**

I implemented the Event handler.
 I also imported the two file request by Microverse

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,13 +1,67 @@
...
+      <div className="keypad">
+        <button type="button" onClick={handleClick} name="AC">AC</button>
+        <button type="button" onClick={handleClick} name="+/-">+/-</button>
+        <button type="button" onClick={handleClick} name="%">%</button>
+        <button type="button" className="highlight" onClick={handleClick} name="/">&divide;</button>
+        <button type="button" onClick={handleClick} name="7">7</button>
+        <button type="button" onClick={handleClick} name="8">8</button>
+        <button type="button" onClick={handleClick} name="9">9</button>
+        <button type="button" className="highlight" onClick={handleClick} name="*">&times;</button>
+        <button type="button" onClick={handleClick} name="4">4</button>
```
Comment (BenMukebo) [inline src/components/Calculator.js]: * [ ]  According to the requirements, you should be passing props from the `Calculator` component to a child component. Kindly create a child component, pass props to it, and rendered it here. I would advise you to create a `Button` component and pass the properties of buttons as props to it. +1 sunglasses

- Note: Smaller components are easy to fix, reusable, and make the app more scalable. :pray: 👍🏻

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,13 +1,67 @@
...
+      </form>
+      <div className="keypad">
+        <button type="button" onClick={handleClick} name="AC">AC</button>
+        <button type="button" onClick={handleClick} name="+/-">+/-</button>
+        <button type="button" onClick={handleClick} name="%">%</button>
+        <button type="button" className="highlight" onClick={handleClick} name="/">&divide;</button>
+        <button type="button" onClick={handleClick} name="7">7</button>
+        <button type="button" onClick={handleClick} name="8">8</button>
+        <button type="button" onClick={handleClick} name="9">9</button>
+        <button type="button" className="highlight" onClick={handleClick} name="*">&times;</button>
```
Comment (BenMukebo) [inline src/components/Calculator.js]: - [ ] I noticed that during the operation of multiplication (X) and division (/), clicking on the equal sign (=) to display the result of the operation, makes all the content of the page disappear. Kindly correcting that 👍🏻

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Bigizi, 

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- Gitflow was correctly used.  ✔️   
- The `Calculator` component imported the `calculate.js` file from the logic directory ✔️ 


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


- [ ] Kindly make sure to add the correct issue link in the contributing sections, Actually your issue link goes to the repo page. let's fix this 👍🏻 

```
## 🤝 Contributing
Feel free to check the [issues page](../../issues/).
```

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Wes-Isaac — APPROVED** [review]
Hi @Bigizi,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Professional README 👍.
- No linter error 💯.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**